src/components/SliderCard/SliderCard.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
+import img from './img.png';
+import React from 'react';
+import Image from 'next/image';
+import './scss/SliderCard.scss';
+
+export const SliderCard: React.FC = () => {
+  return (
+    <div className="card">
+      <Image src={img} alt="img" />
+      <p>

```

Comment:
> Every tag should contains className.

Why is that ?

Reply:
Using classes effectively complements semantic HTML by providing a way to style and manage the presentation of your content without altering the core meaning conveyed by the elements themselves.

In essence, prioritize semantic elements for structure and meaning, and leverage classes for targeted styling and advanced control.

Reply:
I don't think there is a need for a class there, one h2 element. It can be selected by tag, there is no risk that the styles will overlap with another element. The argument for not using classes when it is not necessary is that it positively affects the performance of the website.